### PR TITLE
My Portfolio for Mobile Version

### PR DESCRIPTION
**Pull request by Ashnaali3255**
- Create a logo and add a navigation bar.
- Create a Section for the Introduction.
- Add Social Media Links.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ <title>My Portfolio</title>
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+ integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+ crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+ <header>
+    <h1 class="title">Ashna Ali<span><img src="./Bar.svg"></span></h1>
```
Comment (brainconnect93) [inline index.html:17]: - [ ] I would kindly suggest you wrap the `img` tag holding your hamburger icon inside a `nav` tag and everything should be wrapped inside a `header` tag

Example 👇 
```HTML
<header>
  <a href="#">Ashna Ali</a>
  <nav>
     <img src="./Bar.svg" alt="hamburger icon">
  </nav>
</header>
```

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+## Built With 🔨
+
+- HTML
+- CSS
+- Git & Github
+
+## Live Demo
+
+[Live Demo Link](https://ashnaali3255.github.io/Portfolio/)
+
```
Comment (brainconnect93) [inline README.md:14]: - [ ] It is a best practice to make use of the current README file template given to us by microverse. I would suggest you remove the current template you have and change it to the current one. Kindly follow this[ link](https://github.com/microverseinc/readme-template) to access the latest template

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <ul class="links">
+   <li><a title="linkedIn" href="https://www.linkedin.com/in/ashna-ali-342151255/"><img src="./linkedin.jpeg"></a></li>
+   <li><a title="github" href="https://github.com/Ashnaali3255/Portfolio/tree/Mobile-Branch"><img src="./Github.jpg"></a></li>
+   <li><a title="twitter" href="https://twitter.com/Ashna_Ali1"><img src="./Twitter.jpg"></a></li>
+   <li><a title="Medium" href="https://medium.com/Ashna_Ali1"><img src="./Medium.jpg"></a></li>
+   <li><a title="Angel" href="https://angel.com/Ashna_Ali1"><img src="./Angel.jpg"></a></li>
+    
+
+  </ul>
+ </section>
```
Comment (brainconnect93) [inline index.html:36]: - [ ] I would advise you not to wrap the section inside of a `header` tag. As this is not a good way of writing HTML semantics. Kindly have it separately outside the `header` tag.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  header h1.title {
+    display: flex;
+    justify-content: space-around;
+    color: #6070ff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+  }
```
[the remark this answers is not in the document]
Comment (Ashnaali3255) [reply style.css:18]: I'll try my best.

**Pull request closed without merging**